src/styl/_components/_breakingnews.styl

```diff
@@ -48,6 +48,12 @@
         &-title
             font-size _rem(16px)
             font-weight bold
+            text-overflow ellipsis
+            overflow hidden
+            display -webkit-box !important
```

Comment:
Something off if you have to add an `!important`

Reply:
not really is a recommendation for all browser compatibility issues

Reply:
By whom? Any doc to share?

Reply:
i don't find where i saw it, but : 
https://kiranworkspace.com/ellipsis-to-multiline-text-in-css/
https://caniuse.com/?search=display%3A%20-webkit-box%20

Reply:
is in a conversation in stackoverflow but i don't find it again, so i remove this one and see